config/cesm/config_files.xml

```diff
@@ -238,6 +239,7 @@
       <value component="pop"      >$COMP_ROOT_DIR_OCN/cime_config/config_compsets.xml</value>
       <value component="mpas-o"   >$COMP_ROOT_DIR_OCN/cime_config/config_compsets.xml</value>
       <value component="blom"     >$COMP_ROOT_DIR_OCN/cime_config/config_compsets.xml</value>
+      <value component="norcpm"   >$COMP_ROOT_DIR_ESP/cime_config/config_compsets.xml</value>
```

Comment:
This entry is okay, however, please make sure to adopt a naming convention that helps everyone know when a compset is from NorCPM. Currently, I see [`NHISTNCPM`](https://github.com/BjerknesCPU/norcpm_esp/blob/9de6bf3f32dbbe33cfbd851452120ebff3d1c165/cime_config/config_compsets.xml#L5C12-L5C21) which is a copy of the entry coming into NorESM via NorESMhub/NorESM#565. Will NorCPM have its own standalone (not fully coupled) compsets? If not, this entry is not needed.